### DISCUSSION TITLE
Am new guys

### DISCUSSION THREAD

**Discussion by lakamo-samuel** [community/community#128159] [New to GitHub]
### Hi everyone,

Hey guys am new here I want to input my code to be able to share it can any one help?

**Comment by goldieowner17** [comment]
Hi @Samuel2006-lakamo and welcome!
You'll get a quality response by using the correct category. In this case, you'll want New To GitHub - I've gone ahead and moved your thread for you. :)

**Reply by lakamo-samuel** [reply]
thanks

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Samuel2006-lakamo 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by drdixit** [comment]
Just dropped by to say 👋Hello! @Samuel2006-lakamo

**Reply by lakamo-samuel** [reply]
hello

**Comment by mehtahrishi** [comment]
initialize git commands in your workspace to push code and code files in your repo.

**Reply by lakamo-samuel** [reply]
done

**Comment by ajverse** [comment]
You can Use Hugging Face repo also and there is a VScode extension available for live sharing of your code files. If you can, then go with containerization with Docker.

**Comment by anshkumar2311** [comment]
Welcome to GitHub and the GitHub Community  @Samuel2006-lakamo  ! 😊 Don't worry, we all start as beginners. Take your time to learn, explore, and have fun. If you ever need help, feel free to ask. You’ll get better day by day!

**Reply by lakamo-samuel** [reply]
thanks man

**Comment by Shobhit21Mishra** [comment]
Hi and welcome! 👋 Great to have you here.

If you want to share your code, you have a few options depending on what you're trying to do:

Directly in a Discussion or Comment:

Wrap your code in triple backticks (```) to format it properly.
Example:
```python
print("Hello, world!")
```

This will display it as a clean, formatted code block.

Use a GitHub Gist:

Go to https://gist.github.com/

Paste your code there, give it a filename, and save it.

Share the link here for others to view.

If It’s a Project:

Create a repository on your GitHub account.

Push your code to the repository.

**Comment by CustomRomHarsh** [comment]
> ### Hi everyone,
> Hey guys am new here I want to input my code to be able to share it can any one help?

👋Hello! Welcome to GitHub and the GitHub Community @Samuel2006-lakamo ..

**Reply by lakamo-samuel** [reply]
thanks

**Comment by Jcarrasco96** [comment]
Welcome to GitHub! 😊 Here’s how to share your code:

Upload Files Directly:

Go to your repository.

Click "Add file" → "Upload files".

Drag & drop your files or click to browse.

Use Git Commands (if you have Git installed):
git add .                          # Stages all files
git commit -m "Your message here"  # Commits changes
git push origin main               # Pushes to GitHub

Need more details? Check GitHub’s [Quickstart Guide](https://docs.github.com/en/get-started/quickstart).

Let us know if you get stuck! 🚀

**Reply by lakamo-samuel** [reply]
thanks this helps alot

**Comment by IT22194930** [comment]
> ### Hi everyone,
> Hey guys am new here I want to input my code to be able to share it can any one help?

Hii Samuel !

**Reply by lakamo-samuel** [reply]
hello
